apps/vc-api/docs/tutorials/consent-and-issuance-credential-tutorial.md

```diff
@@ -0,0 +1,939 @@
+
+In the example below, we will issue a permanent residency card to a user we will call "resident". We use the context "https://w3id.org/residentship/v1" which describes the data of a VC of type "PermanentResidentCard". As second credential we will use a consent credential (self signed) by the user (holder).
+
+### Business workflows
+
+#### Issuance Business workflow
+1. For the resident credential, the authority portal authenticates the current user as a known resident.
+1. The resident requests a "PermanentResidentCard" credential on the autority portal.
+1. The autority portal displays a QR code for the citizen to scan with his mobile wallet.
+1. The citizen's mobile wallet provides the autority portal with a presentation to prove control over a DID.
+1. The authority portal reviews the presentation and issues a "PermanentResidentCard" credential containing the resident's information.
+1. The citizen's mobile wallet contacts the autority portal again to receive the "PermanentResidentCard".
+1. The user issues a self signed consent Verifiable Credential as the second credential.
+
+#### Presentation Business workflow
+1. The authority portal displays a QR code requesting a presentation containing the "PermanentResidentCard" and "ConsentCredential" credential.
+1. The resident scans the QR code with their mobile wallet and prompts the user for permission to present the "PermanentResidentCard" and "ConsentCredential" credential.
+1. The resident confirms and the presentation is submitted to the authority portal who can then authorize the resident.
+
+## Technical overview
+From a technical point of view, in this tutorial, we have access to the server API but no mobile wallet is available. So we will use the server API for both roles of the portal and the resident's mobile wallet.
+
+### Technical workflows
+
+#### 1. Credential Issuance workflow
+- [1.1  [Resident] Create a DID](#11-resident-create-a-did)
+- [1.2 [Resident] Obtain a ResidentCard credential](#12-obtain-permanentresidentcard-credential)
+- [1.3 [Resident] Issue a self-signed credential](#13-resident-issue-a-self-signed-credential)
+
+#### 2. Presentation workflow
+- [2.1  [Verifier] Configure Credential Exchange](#21-verifier-configure-credential-exchange)
+- [2.2  [Verifier] Provide an exchange invitation to the resident](#22-verifier-provide-an-exchange-invitation-to-the-resident)
+- [2.3  [Resident] Initiate the presentation exchange](#23-resident-initiate-the-presentation-exchange)
+- [2.4  [Resident] Create the required presentation](#24-resident-create-the-required-presentation)
+- [2.5  [Resident] Continue the exchange](#25-resident-continue-the-exchange)
+- [2.6 [Verifier] Act on Submitted Presentation](#26-verifier-act-on-submitted-presentation)
```

Comment:
The links are broken here. I think it because the actual steps are using `3.` and not `2.`